### PR TITLE
[GLUTEN-2139][DOC] Document Gluten UI

### PR DESCRIPTION
**Pull request by ulysses-you**
## What changes were proposed in this pull request?
Add Gluten UI docs

(Fixes: \#2139)

## How was this patch tested?

N/A

### CODE REVIEW

**Comment by ulysses-you** [comment]
cc @PHILO-HE @zhouyuan

File: docs/get-started/Velox.md [hunk 1]
```diff
@@ -595,3 +595,34 @@ Both Parquet and ORC datasets are sf1024.
 # External reference setup
 
 TO ease your first-hand experience of using Gluten, we have set up an external reference cluster. If you are interested, please contact Weiting.Chen@intel.com.
+
+# Gluten UI
+
+## Gluten event
+
+Gluten provide two events `GlutenBuildInfoEvent` and `GlutenPlanFallbackEvent`:
```
Comment (PHILO-HE) [inline docs/get-started/Velox.md:603]: Nit: provides

File: docs/get-started/Velox.md [hunk 2]
```diff
@@ -595,3 +595,34 @@ Both Parquet and ORC datasets are sf1024.
...
+
+- GlutenBuildInfoEvent, it contains the Gluten build information so that we are able to be aware of the environment when doing some debug.
+  It includes `Java Version`, `Scala Version`, `GCC Version`, `Gluten Version`, `Spark Version`, `Hadoop Version`, `Gluten Revision`, `Backend`, `Backend Revision`, etc.
+
+- GlutenPlanFallbackEvent, it contains the fallback information for each query execution.
+  Note, If the query execution is in AQE, then Gluten will post it for each stage.
```
Comment (PHILO-HE) [inline docs/get-started/Velox.md:609]: Nit: If => if

**Review by PHILO-HE — COMMENTED** [review]
Looks quite clear! Just two trivial comments. Thanks!

**Review by PHILO-HE — APPROVED** [review]

**Pull request merged**